ui/app/components/secure-variable-form.js

```diff
@@ -75,10 +111,16 @@ export default class SecureVariableFormComponent extends Component {
 
   @action
   async save(e) {
-    e.preventDefault();
+    if (e.type === 'submit') {
```

Comment:
`thought`:  This does highlight that this function is now being used to: autosave, save on click and update on click. The function API and logic might benefit from being explicit about this because a refactoring for one use case to this function could break the other 2 down the line.